dpnp/backend/include/dpnp_iface_fptr.hpp

```diff
@@ -127,6 +127,9 @@ enum class DPNPFuncName : size_t
     DPNP_FN_MINIMUM,                  /**< Used in numpy.minimum() implementation  */
     DPNP_FN_MODF,                     /**< Used in numpy.modf() implementation  */
     DPNP_FN_MULTIPLY,                 /**< Used in numpy.multiply() implementation  */
+    DPNP_FN_MULTIPLY_ARRAY_ARRAY,     /**< Used in numpy.multiply(array, array) implementation  */
+    DPNP_FN_MULTIPLY_ARRAY_SCALAR,    /**< Used in numpy.multiply(array, scalar) implementation  */
+    DPNP_FN_MULTIPLY_SCALAR_ARRAY,    /**< Used in numpy.multiply(scalar, array) implementation  */
```

Comment:
You don't need all of them on public API. Only `DPNP_FN_MULTIPLY` is valuable.

Reply:
Currently I have different types mapping for all the cases `array * array`, `scalar * array`, `array * scalar` to be consistent with `numpy`. So let me know how simultaneously to save the consistency and drop the names.

Reply:
Firstly, there are no `scalar * array` and `array * scalar` cases, you have only one for this (`scaclar * array`).
So you don't have different algos for `scalar * array` and `array * scalar` cases.

Secondly, you have the same "different types mapping" for all `lhs * rhs`. 

I don't understand what exactly you cannot do. Try to describe the problem in more detail.
There is no rocket scinece here, see also prev. comments.

Reply:
I don't have the same "different types mapping" for all `lhs * rhs`.

Reply:
Moreover `scalar * array` and `array * scalar` cases have different types mapping.

Reply:
> I don't have the same types mapping for all `lhs * rhs`.

Try to give more details for your issue.
This is really no big problem, just extend the general one. (or there a lot of solutions for that)